src/scrapers/base-beinleumi-group.ts

```diff
@@ -273,6 +276,14 @@ async function fetchAccounts(page: Page, startDate: Moment) {
   return accounts;
 }
 
+async function getCurrentBalance(page: Page) {
```

Comment:
can you move this above the function which calls it?

Reply:
I can but In my opinion it's good practice for clean code to put the higher-level function below the lower level, because then you can read the code from top to bottom.
reading first the lower level implementation makes no sense on its own because there is no context. so you always want to read the higher-level code first, and only then dig down to the lower level implementations (the sub-function it triggers)

Reply:
@LiranBri that's a matter of taste 😉  plus that's how we have the lint rules set up for the entire project at the moment

Reply:
IMO its a bad practice, but your project, your call :)
fixed